feature-libs/cart/package.json

```diff
@@ -39,7 +39,7 @@
     "@ngrx/effects": "^13.0.2",
     "@ngrx/store": "^13.0.2",
     "@spartacus/core": "4.1.0-next.0",
-    "@spartacus/schematics": "4.1.0-next.0",
+    "@spartacus/schematics": "4.1.0",
```

Comment:
this shouldn't be here. This is related to the past message to revert back the changes done when you attempted to manually run schematics through verdaccio.

P.S as with other package changes from other features.
-----
How to fix? 

1. go to `projects/schematics` and in `package.json`, change the version back to `4.1.0-next.0`. 
2. manually delete `dependencies.json` from `projects/schematics/src` 
3. run `yarn config:update` from root